src/tab-active.js

```diff
@@ -1,19 +1,20 @@
-function wikiaJWPlayerHandleTabNotActive(playerInstance) {
-	function canPlayVideo() {
-		return !document.hidden && (['playing', 'paused', 'complete'].indexOf(
-			playerInstance.getState()) === -1 || pausedOnRelated);
-	}
-
+function wikiaJWPlayerHandleTabNotActive(playerInstance, willAutoplay) {
 	var pausedOnRelated = false;
 
-	playerInstance.on('playerStart', function () {
-		document.addEventListener('visibilitychange', function () {
-			if (canPlayVideo()) {
+	function playVideoOnTabSwitch() {
+		if (['playing', 'paused', 'complete'].indexOf(playerInstance.getState()) === -1 || pausedOnRelated) {
+			if (willAutoplay) {
 				playerInstance.play();
 				pausedOnRelated = false;
 				playerInstance.trigger('playerResumedByBrowserTabSwitch');
+			} else {
+				playerInstance.setMute(false);
```

Comment:
Why do we need this? IMO for click-to-play videos the mute property in the options object should be set to false (https://github.com/Wikia/jwplayer-fandom/blame/master/src/index.js#L83)

Reply:
unfortunately it does not work in this situation, video starts muted when you click play